src/qt/locale/bitcoin_zh.ts

```diff
@@ -1310,12 +1310,12 @@ Note:  Since the fee is calculated on a per-byte basis, a fee of "100 satoshis p
         <translation>选择以前使用的地址</translation>
     </message>
     <message>
-        <source>The Bitcoin address to send the payment to</source>
-        <translation>支付到的比特币地址</translation>
+        <source>The Dogecoin address to send the payment to</source>
+        <translation>支付到的狗狗币地址</translation>
```

Comment:
I think it would be more understandable to use “用於將付款發送至的Dogecoin地址”

Reply:
My suggestion is to use '狗狗币收款账户地址'

Reply:
What about 狗狗币接受地址 as a literal translation from "Dogecoin receiving address"?

Reply:
> What about 狗狗币接受地址 as a literal translation from "Dogecoin receiving address"?

Yes, it should be more appropriate by using Steve Gan suggestion.

狗狗币接受地址 ,it's literally dogecoin receiving address (address aka home address, which Steve did add on 收款 账户地址（receiving account address)

Reply:
> > I guess we really need to finalize which Chinese to use,as there are 2 version there "Simplify and Traditional Chinese"
> > 
> > 1. Simplify Chinese : mostly in Pinyin ； country involved - China, Singapore, Malaysia & etc
> > 2. Traditional Chinese : zhuyin ；country involved - Taiwan, Hong Kong & etc
> 
> This is what I've tried to do in the translation fixes, please let me know if you spot anything that doesn't fit. I also used simplified for Chinese (unspecified country), since it's more common worldwide.

Yes,i agree with that, as it is common worldwide. I will change traditional chinese to simplify chinese by today. Some of the translation sound wrong,as it literally like auto-translate and it does not sound right for chinese, they might be confuse ,should i adding the comment as well? (i will try to check some chinese games,apps in order to get a better translation)